.github/workflows/tests.yml

```diff
@@ -27,14 +27,17 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
-      - name: Compile contracts
-        run: yarn contracts:compile
```

Comment:
is it the intent to delete this?

Reply:
This should now be done through a `postinstall` script, so it no longer needs to be included in the yml files.

Reply:
added back as postinstall